script/custom_build_functions.sh

```diff
@@ -0,0 +1,20 @@
+function run_cukes {
+  if [ -d features ]; then
+    echo "${PWD}/bin/cucumber"
+
+    if is_mri_192; then
+      # For some reason we get SystemStackError on 1.9.2 when using
+      # the bin/cucumber approach below. That approach is faster
+      # (as it avoids the bundler tax), so we use it on rubies where we can.
+      bundle exec cucumber --strict
+    elif is_jruby; then
+      return 0
```

Comment:
IMO, we shouldn't silently skip a major part of the build.  It should print a message in this branch so it is clear why the cukes didn't run based just on looking at the travis build output.

Reply:
I'm echoing a message now